src/Illuminate/Support/Str.php

```diff
@@ -794,4 +795,39 @@ public static function createUuidsNormally()
     {
         static::$uuidFactory = null;
     }
+
+    /**
+     * Splits the string by delimiter into a collection of stringable objects.
+     *
+     * @param  string  $string
+     * @param  string  $delimiter
+     * @param  int  $limit
+     *
+     * @return \Illuminate\Support\Collection
+     */
+    public static function split($string, $delimiter = ',', $limit = null)
+    {
+        $parameters = array_filter([$delimiter, $string, $limit]);
+        $pieces = call_user_func_array('explode', $parameters);
+
+        return Collection::make($pieces)
+            ->map([Stringable::class, 'make']);
```

Comment:
This could be replaced with `->mapInto(Stringable::class)` which removes the need to add a `make()` method.